discovery-provider/src/queries/get_remixable_tracks.py

```diff
@@ -0,0 +1,61 @@
+from sqlalchemy import desc
+from src.models import Track, Stem
+from src.queries.query_helpers import create_save_repost_count_subquery, \
+    get_current_user_id, populate_track_metadata, add_users_to_tracks, decayed_score
+from src.utils.db_session import get_db_read_replica
+from src.utils import helpers
+
+def get_remixable_tracks(args):
+    """Gets a list of remixable tracks"""
+    current_user_id = get_current_user_id()
+    db = get_db_read_replica()
+    limit = args.get('limit', 25)
+
+    with db.scoped_session() as session:
+        # Subquery to get current tracks that have stems
+        remixable_tracks_subquery = (
+            session.query(Track)
+                .join(Stem, Stem.parent_track_id == Track.track_id)
+                .filter(
+                    Track.is_current == True,
+                    Track.is_unlisted == False,
+                    Track.is_delete == False
+                )
+                .subquery()
+        )
+
+        count_subquery = create_save_repost_count_subquery(session, 'track')
+
+        query = (
+            session.query(
+                remixable_tracks_subquery,
+                count_subquery.c['count'],
+                decayed_score(
```

Comment:
this looks pretty good to me. let's see into the change to fix the decayed_score fn that we had talked about. could be a subsequent PR.

`5 ^ (1-x)` is sensible, the func.least doesn't make any sense there and should be a `greatest`